lib/winston-logstash.js

```diff
@@ -75,6 +74,7 @@ winston.transports.Logstash = Logstash;
 Logstash.prototype.name = 'logstash';
 
 Logstash.prototype.log = function (level, msg, meta, callback) {
+  meta = meta !== null && meta !== undefined && !(meta instanceof Error) ? cycle.decycle(meta) : meta || null;
```

Comment:
cycle.decycle is in winston itself, is there some reason it has to be duplicated here?